stdlib/public/core/StringObject.swift

```diff
@@ -340,7 +376,7 @@ extension _StringObject {
   @inline(__always)
   internal
   init() {
-    self.init(rawBits: UInt64(_StringObject._emptyStringBitPattern))
+    self.init(taggedRawBits: _StringObject._emptyStringBitPattern)
```

Comment:
This won't compile on 32-bit platforms.

Reply:
Right!

Reply:
And by "right" I mean wrong! At least for this code which is inside a `#if`. The inits certainly won't compile, though.

Reply:
🤦‍♂️ Wow, I didn't expand far enough 😊